plugins/generator_docs.rb

```diff
@@ -77,7 +77,7 @@ def generate_docs
       # The order of these functions will determine the order of preference
       # when looking up symbols e.g. double backticks
       # DO NOT CHANGE THE ORDER UNLESS YOU KNOW WHAT YOU ARE DOING
-      generate_docs_c
+      # generate_docs_c
```

Comment:
Is there a reason this has been disabled? We definitely want this one enabled - don't want to be using the preview docs.